clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/AbstractStorageEngine.java

```diff
@@ -354,6 +354,10 @@ public synchronized void close() throws VeniceException {
     closeMetadataPartition();
   }
 
+  public boolean isClosed() {
+    return this.partitionList.isEmpty();
```

Comment:
Can you verify for sparse list `isEmpty` will actually return the correct value you want.

Reply:
Good catch... it will not do what I want... that is kind of a problem with unit tests and mocking... the mocked behavior is not guaranteed to match with reality. I will come up with another way...

Reply:
I added a `nonNullSize` counter in the `SparseConcurrentList` to deal with this...

Reply:
@FelixGV 
Can you write some unit tests against this utility class?

Reply:
Yep, I did that in the latest commit.